src/xg/test/testxg.cpp

```diff
@@ -154,3 +155,8 @@ int test(std::ostream &outStream)
 		return 1;
 	}
 }
+
+int main()
+{
+	return test(std::cout);
+}
```

Comment:
I had `Test.cpp` and `TestMain.cpp` split into separate files because android uses the test lib but not the `main` entry point. I think this change breaks the android build.